numba/errors.py

```diff
@@ -241,6 +241,35 @@ def termcolor():
 https://github.com/numba/numba/issues/new
 """
 
+interpreter_error_info = """
+Unsupported Python functionality was found in the code Numba was trying to
+compile. This error could be due to invalid code, does the code work
+without Numba? (A quick way to check is to temporarily disable Numba JIT by
+setting the `NUMBA_DISABLE_JIT` environment variable to non-zero and then
+rerunning the code).
+
+If the code is valid and the unsupported functionality is important to you
+please file a feature request at: https://github.com/numba/numba/issues/new
+
+To see Python/NumPy features supported by the latest release of Numba visit:
+http://numba.pydata.org/numba-doc/dev/reference/pysupported.html
+and
+http://numba.pydata.org/numba-doc/dev/reference/numpysupported.html
+"""
+
+constant_inference_info = """
+Numba could not make a constant out of something that it decided should be
+a constant. This could well be a current limitation in Numba's internals,
+please either raise a bug report along with a minimal reproducer at:
+https://github.com/numba/numba/issues/new
+
+(if you need help writing a minimal reproducer please see)
+http://matthewrocklin.com/blog/work/2018/02/28/minimal-bug-reports
```

Comment:
The note about minimal reproducers is redundant to instructions in the issue tracker, so I think we can drop these 2 lines from the error message.

Reply:
Agreed. I've removed this from here and the other "help" strings.